hi-ml-azure/src/health_azure/utils.py

```diff
@@ -1459,6 +1460,35 @@ def download_files_from_run_id(
     torch_barrier()
 
 
+def get_driver_log_file_text(run: Run, download_file: bool = True) -> Optional[str]:
+    """
+    Returns text stored in run log driver file.
+
+    :param run: Run object representing the current run.
+    :param download_file: Boolean flag that control if the log file should be downloaded from the run or not.
+    :return: Driver log file text if a file exists, `None` otherwise.
```

Comment:
Double backticks must be used, not single. See https://docutils.sourceforge.io/docs/user/rst/quickref.html#inline-markup